frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf

```diff
@@ -9,29 +9,55 @@
 --%>
 
 <core_rt:if test="${usingProjects != null  && usingProjects.size()>0}">
-    <table class="table info_table" title="Usage overview">
+    <table class="table info_table" style="margin-bottom: 0; border-bottom-width: 0">
```

Comment:
You can define columns in HTML and the data in JS. You don't have to do both in JS.
here's my quick and dirty change that seemingly works:
```diff
diff --git a/frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf b/frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf
index 9ba5813..caa007c 100644
--- a/frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf
+++ b/frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf
@@ -11,15 +11,23 @@
 --%>
 
 <core_rt:if test="${usingProjects != null  && usingProjects.size()>0}">
-    <table class="table info_table" style="margin-bottom: 0; border-bottom-width: 0">
-        <thead>
-        <tr>
-            <th colspan="3">${documentName} is used by the following projects</th>
-        </tr>
-    </table>
+    <%--<table class="table info_table" style="margin-bottom: 0; border-bottom-width: 0">--%>
+        <%--<thead>--%>
+        <%--<tr>--%>
+            <%--<th colspan="3">${documentName} is used by the following projects</th>--%>
+        <%--</tr>--%>
+    <%--</table>--%>
 
     <div id="usingProjectsTableDiv">
         <table id="usingProjectsTable" title="Usage overview" class="table info_table">
+            <thead>
+            <tr>
+                <th colspan="3">${documentName} is used by the following projects</th>
+            </tr>
+            <tr>
+                <th>Project Name</th><th>Group</th><th>Responsible</th>
+            </tr>
+            </thead>
         </table>
     </div>
 
@@ -30,25 +38,23 @@
             var result = [];
 
             <core_rt:forEach items="${usingProjects}" var="project">
-            result.push({
-                "DT_RowId": "${project.id}",
-                "id": '${project.id}',
-                "name": "<sw360:DisplayProjectLink project="${project}"/>",
-                "group": '<sw360:out value="${project.businessUnit}"/>',
-                "responsible": '<sw360:DisplayUserEmail email="${project.projectResponsible}" bare="true"/>'
-            });
+            result.push([
+                "<sw360:DisplayProjectLink project="${project}"/>",
+                '<sw360:out value="${project.businessUnit}"/>',
+                '<sw360:DisplayUserEmail email="${project.projectResponsible}" bare="true"/>'
+            ]);
             </core_rt:forEach>
 
             usingProjectsTable = $('#usingProjectsTable').DataTable({
                 "pagingType": "full_numbers",
                 "data": result,
                 "searching": false,
-                "lengthChange": false,
-                "columns": [
-                    {title: "Project name", data: "name"},
-                    {title: "Group", data: "group"},
-                    {title: "Responsible", data: "responsible"}
-                ]
+                "lengthChange": false//,
+//                "columns": [
+//                    {title: "Project name", data: "name"},
+//                    {title: "Group", data: "group"},
+//                    {title: "Responsible", data: "responsible"}
+//                ]
             });
         }

```